velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -62,6 +95,31 @@ class ContainerRowSerdeTest : public testing::Test,
     allocator_.clear();
   }
 
+  void testCompareWithNulls(
+      const DecodedVector& decodedVector,
+      const std::vector<HashStringAllocator::Position>& positions,
+      const std::vector<std::shared_ptr<ByteStream>>& byteStreams,
```

Comment:
Per comment above, let's remove byteStreams parameter and create ByteStreeams from positions on the fly.

Reply:
Fixed